src/transaction.js

```diff
@@ -600,6 +614,35 @@ const transaction = {
     }
   },
 
+  /**
+   * Execute some verifications if this data can become a transaction
+   * Validates maximum number of inputs and outputs
+   *
+   * @param {Object} data Object with inputs and outputs
+   * {
+   *  'inputs': [{'tx_id', 'index', 'token', 'address'}],
+   *  'outputs': ['address', 'value', 'timelock', 'tokenData'],
+   * }
+   *
+   * @throws {MaximumNumberInputsError} If the tx has more inputs than the maximum allowed
+   * @throws {MaximumNumberOutputsError} If the tx has more outputs than the maximum allowed
+   *
+   * @memberof Transaction
+   * @inner
+   */
+  verifyTxData(data) {
+    const maxNumberInputs = transaction.getMaxInputsConstant();
+    const maxNumberOutputs = transaction.getMaxOutputsConstant();
+
+    if (data.inputs.length > maxNumberInputs) {
```

Comment:
What if `maxNumberInputs` is null? I think it should be impossible, but would it raise an exception?

Reply:
Yes, it would raise an exception before at `getMaxInputsConstant `. This exception is captured in all methods that send a transaction to show a pretty message to the user

Reply:
But where is it raised?

Reply:
The methods `getMaxInputsConstant` and `getMaxOutputsConstant` throw the error in case it's null.

Reply:
Ok. Thanks!